package/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "geonetwork-ui",
+  "version": "2.1.0-dev",
```

Comment:
Will this be updated automatically, when releasing a new version, like in the root `package.json`?

Reply:
Done in https://github.com/geonetwork/geonetwork-ui/pull/676/commits/1a381ea23cdfd1f40bd7759f82bf315fb17e6b43